docs/getstarted/evals.md

```diff
@@ -0,0 +1,221 @@
+
+
+## Evaluation
+
+In this guide, you will evaluate a **text summarization pipeline**. The goal is to ensure that the output summary accurately captures all the key details specified in the text, such as growth figures, market insights, and other essential information.
+
+`ragas` offers a variety of methods for analyzing the performance of LLM applications, referred to as [metrics](../concepts/metrics/). Each metric requires a predefined set of data points, which it uses to calculate scores that indicate performance.
+
+### Evaluating using a Non-LLM Metric
+
+Here is a simple example that uses `BleuScore` score to score summary
+
+```python
+from ragas import SingleTurnSample
+from ragas.metrics import BleuScore
+
+test_data = {
+    "user_input": "summarise given text\nThe company reported an 8% rise in Q3 2024, driven by strong performance in the Asian market. Sales in this region have significantly contributed to the overall growth. Analysts attribute this success to strategic marketing and product localization. The positive trend in the Asian market is expected to continue into the next quarter.",
+    "response": "The company experienced an 8% increase in Q3 2024, largely due to effective marketing strategies and product adaptation, with expectations of continued growth in the coming quarter.",
+    "reference": "The company reported an 8% growth in Q3 2024, primarily driven by strong sales in the Asian market, attributed to strategic marketing and localized products, with continued growth anticipated in the next quarter."
+}
+metric = BleuScore()
+test_data = SingleTurnSample(**test_data)
+metric.single_turn_score(test_data)
+```
+
+Output
+```
+0.137
+```
+
+Here we used:
+
+- A test sample containing `user_input`, `response` (the output from the LLM), and `reference` (the expected output from the LLM) as data points to evaluate the summary.
+- A non-LLM metric called `BleuScore`.
+
+
+As you may observe, this approach has two key limitations:
+
+- **Time-Consuming Preparation:** Evaluating the application requires preparing the expected output (`reference`) for each input, which can be both time-consuming and challenging.
+
+- **Inaccurate Scoring:** Even though the `response` and `reference` are similar, the output score was low. This is a known limitation of non-LLM metrics like `BleuScore`. 
+
+
+!!! info
+    A **non-LLM metric** refers to a metric that does not rely on an LLM for evaluation.
+
+To address these issues, let's try an LLM-based metric.
+
+
+### Evaluating using a LLM based Metric
+
+
+**Choose your LLM**
+--8<--
+choose_evaluator_llm.md
+--8<--
+
+**Evaluation**
+
+
+Here we will use [`AspectCritic`](../concepts/metrics/available_metrics/aspect_critic.md), which an LLM based metric that outputs pass/fail given the evaluation criteria.
```

Comment:
here, even though it is linked - it is not visible because of the backticks
![image](https://github.com/user-attachments/assets/cb64e0d7-1528-4fdb-88a4-7df5b2b0ae0e)